packages/devtools/test/schema.test.ts

```diff
@@ -757,6 +757,14 @@ describe('schema', () => {
         fields: [{
           path: ["idValue"]
         }],
+      )
+      @createIndex(
+        name: "id_required_index",
```

Comment:
Can you add a test case for the error thrown when the index name already exists please?